apps/aecore/src/aec_pow.erl

```diff
@@ -131,11 +131,11 @@ target_to_difficulty(Th) ->
 
 -spec pick_nonce() -> integer().
 pick_nonce() ->
-    rand:uniform(?NONCE_RANGE) band 16#7fffffff.
+    rand:uniform(?NONCE_RANGE) band ?MAX_NONCE.
 
 -spec next_nonce(integer()) -> integer().
 next_nonce(N) ->
```

Comment:
If we know that N =< ?MAX_NONCE it is more efficient to write this function as
```erlang
next_nonce(?MAX) -> 0;
next_nonce(N) -> N + 1. 
```
but that might not always be the case...